extensions/transform/README.md

```diff
@@ -53,4 +53,36 @@ outcome       https://your-url.com/hello-world
 
 ## Feet and inches to inches
 
-Converts a measurement in feet and inches to just inches.
\ No newline at end of file
+Converts a measurement in feet and inches to just inches.
+
+## Combine date and time
+
+Combines a reference date with a time string to create a complete datetime in UTC format. This action supports two input formats for the time string:
+
+1. **Simple time format (HH:mm:ss)**: Combines the reference date with the specified time in ISO format (e.g., "14:30:00" not "2PM")
+2. **Full datetime with timezone**: Accepts ISO8601 datetime strings with timezone offsets and converts them to UTC
+
+**Examples:**
+
+**Example 1 - Simple time combination:**
+```
+Reference date: 2025-09-13
+Time string: 17:45:00
+Result: 2025-09-13T17:45:00Z
+```
+
+**Example 2 - Timezone-aware datetime:**
+```
+Reference date: 2025-09-06 (ignored when using full datetime)
+Time string: 2025-09-06T15:34:44+02:00
+Result: 2025-09-06T13:34:44Z (converted from +02:00 to UTC)
```

Comment:
The date part of the reference date should take presedence over the date part of the timestring, if present.

Example:
Reference date: 2025-09-01
Time string: 2025-09-06T15:34:44+02:00
Result: 2025-09-01T13:34:44Z